CONTRIBUTING.md

```diff
@@ -46,6 +46,8 @@ this:
  3. **Commit** changes to your own feature branch
  4. **Push** your work back up to your forked repository
  5. Submit a **Pull request** (PR) so that we can review your changes
+ 
+ ⚠ Pull requests without any activity for at least 14 days will be closed. To avoid this, please reply when asked for more information or update your PR in case in didn't meet the requirements (e.g. failed checks).
```

Comment:
Please be consistent in the line length with the rest of the file. I think the maximum length is ~100 characters. A possible exception are URLs or other language constructs which have to be in a single line.